src/Table/Table.stories.jsx

```diff
@@ -77,4 +88,24 @@ storiesOf('Table', module)
       caption="Famous Internet Cats"
       className={['table-responsive']}
     />
-  ));
+  ))
+  .add('sortable', () => {
```

Comment:
I wanted to give an example of an actual sortable table here, so I had to do some hocus-pocus to actually sort the data here.